### DISCUSSION TITLE
npm i @babel/traverse

### DISCUSSION THREAD

**Discussion by NishithHM** [npm/feedback#348] [CLI] [answered]
Hi Folks,

We are not able to download the above mentioned library, please have a look

Steps to reproduce
Do npm i @babel/traverse on project root folder, 

Error Output
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @babel/generator@^7.13.16.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of '@babel/traverse'
npm ERR! notarget 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nishithhm/.npm/_logs/2021-04-22T05_36_11_452Z-debug.log

I see @babel/traverse is present in the npm registry (https://www.npmjs.com/package/@babel/traverse)
[2021-04-22T05_36_11_452Z-debug.log](https://github.com/npm/feedback/files/6355760/2021-04-22T05_36_11_452Z-debug.log)

**Comment by MylesBorins** [comment, marked as answer]
Hey NishithHM. This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!

FWIW I attempted to install @babel/traverse and did not run into any problems, it looks like it was recently published so perhaps you ran into a race condition?

**Reply by MylesBorins** [reply]
I'm locking the conversation as multiple folks are coming by looking for technical support which we do not offer through this channel

**Comment by adityakmehta** [comment]
Getting Error while doing yarn install.
Couldn't find any versions for "@babel/helper-compilation-targets" that matches "^7.13.16"
? Please choose a version of "@babel/helper-compilation-targets" from this list: (Use arrow keys)

7.13.13
7.13.10
7.13.8
7.13.0
7.12.17
7.12.16
7.12.13
7.12.5
7.12.1
7.12.0
7.10.4
7.10.2
7.10.1
7.10.0
7.9.6
7.8.7
7.8.6
7.8.4
7.8.3
7.8.1
7.8.0

**Reply by ljharb** [reply]
Perhaps you have a private registry that’s not up to date? https://www.npmjs.com/package/@babel/helper-compilation-targets/v/7.13.16

**Comment by alexis-puska** [comment]
Error with 7.15.4 
No matching version found for @babel/parser@^7.15.4
